src/Manager/ArtistManager.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+
+namespace App\Manager;
+
+
+use App\Controller\Artist;
+use App\Entity\ArtistEntity;
+use App\Mapper\AutoMapper;
+use Doctrine\ORM\EntityManagerInterface;
+use Exception;
+
+class ArtistManager
+{
+    private $entityManager;
+
+
+    public function __construct(EntityManagerInterface $entityManagerInterface)
+    {
+        $this->entityManager = $entityManagerInterface;
+    }
+
+    public function create($artist)
+    {
+        $artistEntity=new ArtistEntity();
+        $mapper=new AutoMapper();
```

Comment:
if you pass the class name to the mapper , you can create the class inside it , 
what do you think ?